swan-lake/learn/coding-conventions.md

```diff
@@ -138,17 +142,19 @@ function setName(string name)
 ```ballerina
 
 function func1() {
-    if (true) {}
+    if (true) {
+    }
 }
 
 ```
 
-* If a block is empty, do not keep spaces in between the opening and closing braces.
+* If a block is empty, keep a single newline in between the opening and closing braces.
```

Comment:
As far as I remember this depends on the type of block, shall we recheck this?

Reply:
Yes, it does. Let's update this after the Formatting Guidelines are finalized. I'll revert these changes for the moment.

Reply:
This will be addressed separately after the formatting guidelines doc is finalized.

Reply:
Addressed in #1724